h/static/scripts/guest.coffee

```diff
@@ -17,7 +17,7 @@ module.exports = class Annotator.Guest extends Annotator
     ".annotator-adder button click":     "onAdderClick"
     ".annotator-adder button mousedown": "onAdderMousedown"
     ".annotator-adder button mouseup":   "onAdderMouseup"
-    "setTool": "onSetTool"
+    ".annotator-highlighter button click": "onHighlighterClick"
```

Comment:
May or may not be simpler and repeat less code if you simply let `onAdderClick` handle both and check `event.target` to determine which button it was. We're already overriding `onAdderClick` in this file, so that's a natural place to do this, especially since all the code is the same between this and `onHighlighterClick` except that one calls `createAnnotation` and the other calls `createHighlight`.